src/marks/auto.js

```diff
@@ -346,7 +348,11 @@ function isReducer(reduce) {
 }
 
 function isUnderspecifiedReduce({value, reduce}) {
-  return value == null && reduce != null && /^count$/i.test(reduce);
+  return value == null && reduce != null && !/^count$/i.test(reduce);
+}
+
+function isPrimitive(values) {
+  return isEvery(values, (d) => ["number", "boolean", "string"].includes(typeof d) || d instanceof Date);
```

Comment:
Couple nits here.

First, isEvery requires values to be an iterable, but mark data can also be an “arrayish” value that is supported by Array.from such as {length: 10}. So, we’d need to handle that case, too. I recommend protecting the call as `isIterable(data) && !isPrimitive(data)` (since any data is not iterable by definition can only contain undefined, as in a sparse array, which we would not consider to be primitive).

Second, we should also treat bigints as primitive. (There’s also the *symbol* type to consider. I guess you could use the _mode_ reducer on non-primitive values? But that’s obscure enough that I wouldn’t worry about it. If you’re doing these things you probably can debug the error.)

Last, I would avoid array.includes(type) in the inner loop. Unless the JavaScript engine is smart, it’ll have to allocate that array afresh for each value, and the call to array.includes is also probably slower than an inline logical expression. It’s better to just write this by hand; see isOrdinal for an example.